lxd/provider.go

```diff
@@ -184,6 +184,13 @@ func Provider() terraform.ResourceProvider {
 				Description: descriptions["lxd_refresh_interval"],
 				Default:     "10s",
 			},
+
+			"project": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: descriptions["lxd_project"],
+				Default:     "default",
```

Comment:
This will probably break backwards compatibility with LXD releases that don't support "project". I recommend not setting a default.